compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/repo/HomeBaloRepo.java

```diff
@@ -132,13 +133,16 @@ public Patten calculate(PackageID moduleID) {
      * @throws IOException Error when getting the list of version of the module folder.
      */
     private Optional<Path> getLatestBaloFile(Path moduleFolder) throws IOException {
-        return Files.list(moduleFolder)
-                                        .map(SortablePath::new)
-                                        .filter(SortablePath::valid)
-                                        .sorted(Comparator.reverseOrder())
-                                        .limit(1)
-                                        .map(SortablePath::getPath)
-                                        .findFirst();
+        Optional<Path> path;
+        try (Stream<Path> stream = Files.list(moduleFolder)) {
```

Comment:
Instead of "stream" shall we use "fileStream". "stream" is too vague for me.

Reply:
Renamed variable.